Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -515,6 +516,7 @@ def types_that_cannot_be_forward_declared():
         'PlatformXR::SessionMode',
         'PlatformXR::VisibilityState',
         'String',
+        'WebCore::ApplePayAMSUIRequest',
```

Comment:
Rather than adding all these types as not being able to forward-declared, we should probably suppress the warning in the generated code. Since message types are only instantiated temporarily to encode / decode bytes, they're quite a bit like function arguments. There is no inherent unsafely in using raw references or pointers there.

Reply:
Ok. This indeed would be more maintainable / scalable in the long run.

Reply:
@rniwa So I understand you mean introducing a macro for FowardDeclChecker at https://searchfox.org/wubkat/rev/43260faced156fc0872fccd2b09f39c0dc514f08/Source/WTF/wtf/Compiler.h#527 and making sure the script uses it in various places in the generated code, right?